olm-catalog/serverless-operator/project.yaml

```diff
@@ -41,17 +41,17 @@ dependencies:
     net_istio_artifacts_branch: release-v1.12
     redhat-knative-istio-authz-chart: 1.34.0
     maistra: 2.4-latest
-    eventing: knative-v1.12
+    eventing: knative-v1.14
```

Comment:
For the next release we call Knative 1.14 genuinely 1.14?

Reply:
Yes! :)

Reply:
I assume so. We only used 1.12 (which was actually 1.13), because we wanted the 1.13 bits but SO didn't allow us that time really to skip versions.

Reply:
Thanks, sounds good! :)